Microsoft.Toolkit.Uwp.Services/Services/Twitter/TwitterOAuthRequest.cs

```diff
@@ -58,17 +59,17 @@ internal class TwitterOAuthRequest
         /// <returns>awaitable task</returns>
         public async Task ExecuteGetStreamAsync(Uri requestUri, TwitterOAuthTokens tokens, TwitterStreamCallbacks.RawJsonCallback callback)
         {
-            using (var request = new HttpHelperRequest(requestUri, HttpMethod.Get))
+            using (var request = new HttpRequestMessage(HttpMethod.Get, requestUri))
             {
                 var requestBuilder = new TwitterOAuthRequestBuilder(requestUri, tokens);
 
-                request.Headers.Authorization = HttpCredentialsHeaderValue.Parse(requestBuilder.AuthorizationHeader);
+                request.Headers.Authorization = AuthenticationHeaderValue.Parse(requestBuilder.AuthorizationHeader);
 
-                using (var response = await HttpHelper.Instance.GetInputStreamAsync(request).ConfigureAwait(false))
+                using (var response = await client.SendAsync(request).ConfigureAwait(false))
```

Comment:
Technically, `HttpClient.ReadAsStreamAsync` is doing exactly what the code below is doing. It will just send the GET request to the specified URI and then read the content as Stream. We can reduce the code down to that call if you would prefer, i was just keeping everything intact from the original code without changing much for easier reviewing. 

https://github.com/dotnet/corefx/blob/master/src/System.Net.Http/src/System/Net/Http/HttpClient.cs